TemplateApplication/Schedule/EventContextView.swift

```diff
@@ -64,21 +64,18 @@ struct EventContextView: View {
 
 
 #if DEBUG
-struct EventContextView_Previews: PreviewProvider {
-    private static let task = TemplateApplicationScheduler.socialSupportTask
-
+#Preview {
```

Comment:
I think that a size that fits mechanism would be a good fit here as this is a view that doesn't really fill the complete screen.